gridcapa-swe-commons/src/main/java/com/farao_community/farao/gridcapa_swe_commons/shift/SweNetworkShifter.java

```diff
@@ -134,12 +107,8 @@ public void shiftNetwork(double stepValue, Network network) throws GlskLimitatio
                 double mismatchEsFr = targetExchanges.get(ES_FR) - bordersExchanges.get(ES_FR);
 
                 // Step 3: Checks balance adjustment results
-                if (Math.abs(mismatchEsPt) < toleranceEsPt && Math.abs(mismatchEsFr) < toleranceEsFr) {
-                    String logShiftSucceded = String.format("[%s] : Shift succeed after %s iteration ", direction, ++iterationCounter);
-                    LOGGER.info(logShiftSucceded);
-                    businessLogger.info("Shift succeed after {} iteration ", ++iterationCounter);
-                    String msg = String.format("Exchange ES-PT = %.2f , Exchange ES-FR =  %.2f", bordersExchanges.get(ES_PT), bordersExchanges.get(ES_FR));
-                    businessLogger.info(msg);
+                if (isShiftSucceed(targetExchanges, bordersExchanges)) {
```

Comment:
instead of passing **targetExchanges** and **bordersExchanges** maps to the isShiftSucceed method, use the already computed **mismatchEsFr** and **mismatchEsPt**